### ISSUE TITLE
Runtime Error 'number already in use' in Tunnel Community

### ISSUE THREAD

**Issue by xoriole** [Tribler/tribler#3405]
From reporter 
```python
Traceback (most recent call last):
File "TriblerGUI\event_request_manager.py", line 99, in on_read_data
RuntimeError: Unhandled Error
Traceback (most recent call last):
File "lib\multiprocessing\process.py", line 114, in run

File "TriblerGUI\core_manager.py", line 65, in start_tribler_core

File "lib\site-packages\twisted\internet\base.py", line 1243, in run

File "lib\site-packages\twisted\internet\base.py", line 1252, in mainLoop

--- <exception caught here> ---
File "lib\site-packages\twisted\internet\base.py", line 878, in runUntilCurrent

File "Tribler\Core\Libtorrent\LibtorrentDownloadImpl.py", line 280, in do_check

File "Tribler\community\tunnel\tunnel_community.py", line 467, in build_tunnels

File "Tribler\dispersy\util.py", line 37, in helper

File "Tribler\community\tunnel\tunnel_community.py", line 449, in do_circuits

File "Tribler\community\tunnel\tunnel_community.py", line 580, in create_circuit

File "Tribler\community\tunnel\tunnel_community.py", line 51, in __init__

File "Tribler\dispersy\requestcache.py", line 20, in __init__

exceptions.RuntimeError: This number is already in use '663943629'
```

**Comment by qstokkink** [comment]
This is a Dispersy/IPv8 issue.

**Comment by qstokkink** [comment]
It seems the `CircuitRequestCache` and `ExtendRequestCache` share the same prefix. So for some reason a `circuit_id` is being created which is being extended?

**Comment by ichorid** [comment]
We have to fix it to have #3590 fixed properly.

**Comment by xoriole** [comment]
Considering end of life of Dispersy and a new similar issue https://github.com/Tribler/tribler/issues/3995 I'm closing this one.
